torch/nn/modules/pixelshuffle.py

```diff
@@ -50,3 +50,45 @@ def forward(self, input: Tensor) -> Tensor:
 
     def extra_repr(self) -> str:
         return 'upscale_factor={}'.format(self.upscale_factor)
+
+
+class PixelUnshuffle(Module):
+    r"""Rearranges elements in a tensor of shape :math:`(*, C, H \times r, W \times r)`
+    to a tensor of shape :math:`(*, C \times r^2, H, W)`. This is the inverse operation
+    of :class:`~torch.nn.PixelShuffle`.
+
+    Note that this function can take inputs with any number of batch dimensions:
```

Comment:
I don't think we need this note and should reinforce that `*` is zero or more batch dimensions, since that pattern appears consistently in the docs.